.eslintrc

```diff
@@ -12,7 +12,7 @@
   "rules": {
     "eol-last": 1,
     "no-unused-vars": [1],
-    "max-len": [1, 160, 4],
+    "max-len": [0],
```

Comment:
is prettier using these rules? i think if we set a max length, it would help some of the formatting I already don't like

Reply:
No, it's to stop lint handling things that are done automatically. Prettier is deterministic based on the AST.

Reply:
Related to above, does Prettier automatically add a newline to end of each file? If so the `eol-last` rule can probably be removed.